### ISSUE TITLE
Mounting fails with version 24+really1.3.4-2build2 on Ubuntu Noble

### ISSUE THREAD

**Issue by anderoo** [koverstreet/bcachefs-tools#298]
Hello, it seems like there's some issues with the latest version on Noble as
mounting fails with an unknown filesystem error:

```
> bcachefs version
1.3.3

> bcachefs format /dev/nvme1n1
> mount -t bcachefs /dev/nvme1n1 /var/lib/example
mount: /var/lib/example: unknown filesystem type 'bcachefs'.
       dmesg(1) may have more information after failed mount system call.
```

**Comment by koverstreet** [comment]
that means your kernel doesn't support bcachefs

**Comment by anderoo** [comment]
> that means your kernel doesn't support bcachefs

Wasn't bcachefs support released in 6.7? Ubuntu Noble is 6.8
